src/client.ts

```diff
@@ -206,6 +206,33 @@ export class Client {
   geolocate(request: GeolocateRequest): Promise<GeolocateResponse> {
     return geolocate(request, this.axiosInstance);
   }
+  /**
+   * An example use of this function.
+   *
+   * ```javascript
+   * import { Client } from '@googlemaps/google-maps-services-js';
+   * 
+   * const getFirstResult = async (theAddress) => {
```

Comment:
can probably remove this wrapping function since the status check isn't necesary